projects/core/src/i18n/i18n.module.ts

```diff
@@ -1,11 +1,33 @@
-import { ModuleWithProviders, NgModule } from '@angular/core';
+import { APP_INITIALIZER, ModuleWithProviders, NgModule } from '@angular/core';
+import { map, take } from 'rxjs/operators';
+import { ConfigInitializerService } from '../config/config-initializer/config-initializer.service';
+import { provideDefaultConfig } from '../config/config-providers';
 import { defaultI18nConfig } from './config/default-i18n-config';
 import { CxDatePipe } from './date.pipe';
 import { i18nextProviders } from './i18next/i18next-providers';
 import { I18nextTranslationService } from './i18next/i18next-translation.service';
 import { TranslatePipe } from './translate.pipe';
 import { TranslationService } from './translation.service';
-import { provideDefaultConfig } from '../config/config-providers';
+
+export function initI18nConfig(configInit: ConfigInitializerService) {
+  const result = () =>
+    configInit
+      .getStable('context.i18n')
```

Comment:
There is no config `context.i18n`. Typo?
```suggestion
      .getStable('context.language')
```

Reply:
Fix typo `context.i18n` to `context.language`

Reply:
Done